config/crossSections13p6TeV.yaml

```diff
@@ -0,0 +1,718 @@
+
+WtoLNu_0J_amcatnloFXFX:
+  crossSec: 55760.0
+  reference: XSDB WtoLNu-2Jets_0J_TuneCP5_13p6TeV_amcatnloFXFX-pythia8 modifiedOn 2024-04-03 20:09:44
+  unc: 75.39
+WtoLNu_1J_amcatnloFXFX:
+  crossSec: 9529.0
+  reference: XSDB WtoLNu-2Jets_1J_TuneCP5_13p6TeV_amcatnloFXFX-pythia8 modifiedOn 2024-04-03 20:09:46
+  unc: 56.7
+WtoLNu_1J_madgraphMLM:
+  crossSec: 9084.0
+  reference: XSDB WtoLNu-4Jets_1J_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 20:10:09
+  unc: 24.69
+WtoLNu_2J_amcatnloFXFX:
+  crossSec: 3532.0
+  reference: XSDB WtoLNu-2Jets_2J_TuneCP5_13p6TeV_amcatnloFXFX-pythia8 modifiedOn 2024-04-03 20:09:47
+  unc: 34.81
+WtoLNu_2J_madgraphMLM:
+  crossSec: 2925.0
+  reference: XSDB WtoLNu-4Jets_2J_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 20:10:12
+  unc: 8.555
+WtoLNu_3J_madgraphMLM:
+  crossSec: 861.7
+  reference: XSDB WtoLNu-4Jets_3J_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 20:10:13
+  unc: 2.624
+WtoLNu_4J_madgraphMLM:
+  crossSec: 416.5
+  reference: XSDB WtoLNu-4Jets_4J_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 20:10:17
+  unc: 1.28
+WtoLNu_amcatnloFXFX:
+  crossSec: 3*(9013.3 + 12128.4)
+  reference: https://twiki.cern.ch/twiki/bin/viewauth/CMS/MATRIXCrossSectionsat13p6TeV
+  unc: +1.2%% -1.1%% +-0.8%% (e-nu) +1.1%% -1.4%% +-0.7%% (e+nu)
+WtoLNu_madgraphMLM:
+  crossSec: 3*(9013.3 + 12128.4)
+  reference: https://twiki.cern.ch/twiki/bin/viewauth/CMS/MATRIXCrossSectionsat13p6TeV
+  unc: +1.2%% -1.1%% +-0.8%% (e-nu) +1.1%% -1.4%% +-0.7%% (e+nu)
+WtoLNu_MLNu-0to120_HT-40To100_madgraph:
+  crossSec: 4254.0
+  reference: XSDB WtoLNu-4Jets_MLNu-0to120_HT-40to100_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 20:10:25
+  unc: 12.08
+WtoLNu_MLNu-0to120_HT-100To400_madgraph:
+  crossSec: 1626.0
+  reference: XSDB WtoLNu-4Jets_MLNu-0to120_HT-100to400_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 20:10:18
+  unc: 4.823
+WtoLNu_MLNu-0to120_HT-400To800_madgraph:
+  crossSec: 59.99
+  reference: XSDB WtoLNu-4Jets_MLNu-0to120_HT-400to800_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 20:10:23
+  unc: 0.1803
+WtoLNu_MLNu-0to120_HT-800To1500_madgraph:
+  crossSec: 6.23
+  reference: XSDB WtoLNu-4Jets_MLNu-0to120_HT-800to1500_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 20:10:28
+  unc: 0.01876
+WtoLNu_MLNu-0to120_HT-1500To2500_madgraph:
+  crossSec: 0.4477
+  reference: XSDB WtoLNu-4Jets_MLNu-0to120_HT-1500to2500_TuneCP5_13p6TeV_madgraphMLM-pythia8 modifiedOn 2024-04-03 20:10:20
+  unc: 0.00135
+WToMuNu_M-200:
+  crossSec: 7.693
+  reference: XSDB WtoMuNu_M-200_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 14:05:35
+  unc: 0.0408
+WToTauNu_M-200:
+  crossSec: 7.753
+  reference: XSDB WtoNuTau_M-200_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 14:05:40
+  unc: 0.04088
+
+WW:
+  crossSec: 80.23
+  reference: XSDB WW_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 14:02:32
+  unc: 0.3733
+WWto2L2Nu:
```

Comment:
```suggestion
WWto2L2Nu_powheg:
```

Reply:
I do not see why this line we want to add _powheg back in, where the previous we saw powheg == amcat and we removed any identification